### PR TITLE
'solution'

### PR DESCRIPTION
**Pull request by alexandra-halytska**

### CODE REVIEW

**Review by Oleksandr563487953 — APPROVED** [review]

File: tests/test_main.py [hunk 1]
```diff
@@ -1 +1,26 @@
...
+    assert cryptocurrency_action(current_rate) ==\
+           "Sell all your cryptocurrency"
+
+
+@mock.patch("app.main.get_exchange_rate_prediction")
+def test_prediction_do_nothing(mocked_get_exchange_rate):
+    mocked_get_exchange_rate.return_value = 8
+    current_rate = 8
+    assert cryptocurrency_action(current_rate) ==\
+           "Do nothing"
```
Comment (MasterpieceElbow) [inline tests/test_main.py:26]: Add boundary conditions, you don't check when get_exchange_rate_prediction return value equals exactly 1.05 or 0.95 * value passed in cryptocurrency_action.

**Review by MasterpieceElbow — CHANGES_REQUESTED** [review]

**Review by MasterpieceElbow — APPROVED** [review]
